rust/hyper/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+http = "1.2.0"
+http-body-util = "0.1.2"
+hyper = { version = "1.5.2", features = ["full"] }
+hyper-util = { version = "0.1.10", features = ["full"] }
+tokio = { version = "1", features = ["full"] }
```

Comment:
not needed imho, cause of https://github.com/hyperium/hyper/blob/30f2961e89eb306780d856e6e2c1ee10ffbbafd2/Cargo.toml#L26